### PR TITLE
Update sane reports pdf docker tag

### PR DESCRIPTION
**Pull request by agamm**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23297
related to this revert: https://github.com/demisto/content/pull/7717/files

## Description
Updates the docker tag for the sane-pdf-reports to a new fix version.

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [X] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by ronykoz** [comment]
@agamm i think you still need the release notes here

**Comment by agamm** [comment]
> @agamm i think you still need the release notes here

Your cli tool isn't working for me:
image

What are the manual steps to do it now without breaking things?

**Comment by amshamah419** [comment]
I’ll take a look at the cli issue, but to manually do it:
- bump the `currentVersion` in the `pack_metadata.json`
- Create a markdown file with the name of the new version. Place it under the `ReleaseNotes` dir. `.` is replaced with `_` in the file name.
-  use the following template:

```
#### Scripts
##### (script name goes here)
- (notes go here)
```

**Comment by agamm** [comment]
> I’ll take a look at the cli issue, but to manually do it:
> 
> * bump the `currentVersion` in the `pack_metadata.json`
> * Create a markdown file with the name of the new version. Place it under the `ReleaseNotes` dir. `.` is replaced with `_` in the file name.
> * use the following template:
> 
> ```
> #### Scripts
> ##### (script name goes here)
> - (notes go here)
> ```

Thanks for the info! Updated.

File: Packs/Base/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Base",
     "description": "The base pack for Cortex XSOAR.",
     "support": "xsoar",
-    "currentVersion": "1.0.13",
+    "currentVersion": "1.0.14",
```
Comment (Bars92) [inline Packs/Base/pack_metadata.json:5]: why is that?

Comment (ronykoz) [reply Packs/Base/pack_metadata.json:5]: yes

**Comment by yaron-libman** [comment]
Reviewed the text.

**Pull request merged**